packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1231,8 +1243,20 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
   /// The unique identifier for this node.
   ///
   /// The root node has an id of zero. Other nodes are given a unique id when
-  /// they are created.
-  final int id;
+  /// they are attached.
```

Comment:
I am trying to thing of any issues that could arise from re-attaching a temporarily unattached node under a different ID. Could this cause issues with a11y focus, which is tied to node id?